index.js

```diff
@@ -149,21 +214,26 @@ function createSeries (opts, archs, platforms) {
 module.exports = function packager (opts, cb) {
   var archs = validateList(opts.all ? 'all' : opts.arch, supportedArchs, 'arch')
   var platforms = validateList(opts.all ? 'all' : opts.platform, supportedPlatforms, 'platform')
-  if (!opts.version) return cb(new Error('Must specify version'))
   if (!Array.isArray(archs)) return cb(new Error(archs))
   if (!Array.isArray(platforms)) return cb(new Error(platforms))
 
-  // Ignore this and related modules by default
-  var defaultIgnores = ['/node_modules/electron-prebuilt($|/)', '/node_modules/electron-packager($|/)', '/\.git($|/)']
-  if (opts.ignore && !Array.isArray(opts.ignore)) opts.ignore = [opts.ignore]
-  opts.ignore = (opts.ignore) ? opts.ignore.concat(defaultIgnores) : defaultIgnores
-
-  series(createSeries(opts, archs, platforms), function (err, appPaths) {
-    if (err) return cb(err)
-
-    cb(null, appPaths.filter(function (appPath) {
-      // Remove falsy entries (e.g. skipped platforms)
-      return appPath
-    }))
+  getNameAndVersion(opts, function (err) {
+    if (err) {
+      return cb(err)
+    }
```

Comment:
Ah, standard yelled at me for not having braces, I didn't realize you could do this. Fixed.